blocked-edges/4.18.0-KubeletStartFailingFromRestoreconTimeout.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.18.0
+from: 4[.]17[.].*
+url: https://issues.redhat.com/browse/OCPNODE-3245
+name: KubeletStartFailingFromRestoreconTimeout
+message: |-
+  In some cases, a SELinux restorecon run as a prerequisite of kubelet.service was failing. Now, the restorecon is more targeted.
```

Comment:
maybe we should drop "Now, the restorecon is more targeted", because "Now" isn't all that clear?  Folks who are wondering about the fix can click through to OCPNODE-3245 and see  more context there, and folks who see 4.18.13 recommended without mentioning this risk won't be reading this message.